windpowerlib/modelchain.py

```diff
@@ -438,6 +439,11 @@ def run_model(self, weather_df):
         'wind_speed'
 
         """
+        # Convert data heights to integer. In some case they are strings.
+        weather_df.columns = pd.MultiIndex.from_arrays([
+            weather_df.columns.get_level_values(0),
+            weather_df.columns.get_level_values(1).astype(int)])
```

Comment:
Why `int` and not `float` type? In the provided example the values are integers but that shouldn't be a rule, right?

Reply:
In my opinion the inaccuracy of the model is higher than the decimal digit. Therefore I would always prefer integer. 

But at them moment floats are allowed and it might be easier to allow floats than to forbid them and to explain why.